Makefile

```diff
@@ -131,7 +131,7 @@ test/e2e: cluster/cleanup cluster/cleanup/crds cluster/prepare cluster/prepare/c
 .PHONY: test/functional
 test/functional:
 	# Run the functional tests against an existing cluster. Make sure you have logged in to the cluster.
-	go test ./test/functional
+	go clean -testcache && go test ./test/functional
```

Comment:
Using the `-v` flag is enough:

Form the go doucumentation
```
When 'go test' runs in package list mode, 'go test' caches successful
package test results to avoid unnecessary repeated running of tests. To
disable test caching, use any test flag or argument other than the
cacheable flags. The idiomatic way to disable test caching explicitly
is to use -count=1.
```

Reply:
Just tested, unfortunately `-v` still uses cached results. I had seen the alternative solution of using `-count=1` but this seems more like a workaround to me vs using `go clean -testcache`.